### DISCUSSION TITLE
Update from 2.5.4 to 3.1.3

### DISCUSSION THREAD

**Discussion by martrze** [templaza/astroid-framework#788] [General]
I want to upgrade my astroid zero framework from 2.5.4 to 3.1.3.
I've downloaded the framework and installed by uploading it. There's a message that package was installed successful but nothing more - it's still old version. Also the way by installing it from the web page ends the same.
Can I do it manually by uploading files via FTP? Where exactly? Is there a risk that it will broke the page? 
Is there another way to do it? Or I'm missing something?

**Comment by paurray** [comment]
@martrze 
What is the exact name of the file that you uploaded and installed?

astroid-framework-3.x.x.zip ?
or
astroid-package-v3.x.x.Framework-Template.zip ?

This is something that I also find very confusing so for eg:

here there are 4 assets
https://github.com/templaza/astroid-framework/releases/tag/v3.1.3

and

https://github.com/templaza/astroid-framework/releases/tag/v3.1.1

here there are 5 assets including

astroid-package-v3.1.1.Framework-Template.zip

I find this confusing as well.

You could also try looking in your Extensions Manager  and eyeball Astroid related 
Assets and version numbers...

<img width="960" alt="AstroidcStateOfPlay" src="https://github.com/user-attachments/assets/a1a345c6-a68b-47fd-a4a7-900c76d1b568">

Hope this helps rather than confusing matters!

**Reply by sonvnn** [reply]
Since 3.1.3 you don't need template-package. System will auto-suggest install template packages if you site does not contain any templates

**Reply by paurray** [reply]
@sonvnn Just thinking aloud here but how about auto-suggest install template packages if your site does contain an out of date Astroid Template? As in this use case. Does this make any sense?

**Reply by sonvnn** [reply]
If you have an out-date template. Joomla update system will remind you to update it. Don't worry about it ;)

**Reply by paurray** [reply]
thanks

**Comment by martrze** [comment]
Ok
I've got it. First I've updated to 3.1.1 and then up to 3.1.3.
Thank You.

**Reply by paurray** [reply]
Glad to hear that this worked out for you!
